aea/protocols/generator.py

```diff
@@ -555,6 +582,8 @@ def _setup(self) -> None:
             pure_custom_type: "Custom" + pure_custom_type
             for pure_custom_type in self._all_custom_types
         }
+        if self.protocol_specification.dialogue_config:
```

Comment:
We use the explicit form throughout: `if self.protocol_specification.dialogue_config != {}:

Reply:
Done

Reply:
and all other occurrences